README.md

```diff
@@ -1 +1,122 @@
-# saml-android-app
+# Android SAML Template
+---------
+Author: Daniel Passos (dpassos@redhat.com, daniel@passos.me)     
+Level: Intermediate  
+Technologies: Java, Android, RHMAP, SAML  
+Summary: A demonstration of how to integrate with the [FeedHenry SAML PoC](https://github.com/feedhenry-templates/saml-cloud-app)  
+Community Project : [Feed Henry](http://feedhenry.org/)  
+Target Product: RedHat Mobile Application Platform aka RHMAP    
+Product Versions: RHMAP 3.8.0+   
+Source: https://github.com/feedhenry-templates/pushstarter-android-app  
+Prerequisites: fh-android-sdk : 3.0.+, Android Studio : 1.4.0 or newer, Android SDK : 22+ or newer
+
+## What is it?
+
+This application will subscribe to a push service running in a RHMAP instance. The user can send messages to the device using RHMAP and view them on the device.  
```

Comment:
Couldn't think of a better wording.

Summers Pittman
>>Phone:404 941 4698
>>Java is my crack.


On Fri, Mar 4, 2016 at 12:57 PM, Daniel Passos <notifications@github.com>
wrote:

> In README.md
> <https://github.com/feedhenry-templates/saml-android-app/pull/8#discussion_r55063891>
> :
>
> > -# saml-android-app
> > +# Android SAML Template
> > +---------
> > +Author: Daniel Passos (dpassos@redhat.com, daniel@passos.me)
> > +Level: Intermediate
> > +Technologies: Java, Android, RHMAP, SAML
> > +Summary: A demonstration of how to integrate with the [FeedHenry SAML PoC](https://github.com/feedhenry-templates/saml-cloud-app)
> > +Community Project : [Feed Henry](http://feedhenry.org/)
> > +Target Product: RedHat Mobile Application Platform aka RHMAP
> > +Product Versions: RHMAP 3.8.0+
> > +Source: https://github.com/feedhenry-templates/pushstarter-android-app
> > +Prerequisites: fh-android-sdk : 3.0.+, Android Studio : 1.4.0 or newer, Android SDK : 22+ or newer
> > +
> > +## What is it?
> > +
> > +This application will subscribe to a push service running in a RHMAP instance. The user can send messages to the device using RHMAP and view them on the device.
>
> push service?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/feedhenry-templates/saml-android-app/pull/8/files#r55063891>
> .
>